doc/source/changes.rst

```diff
@@ -8,6 +8,41 @@ introduce differences in the yml configuration file. So it would not be enough t
 version. Often a version upgrade changes or adds parameters which are used.
 
 
+.. _changes-version-1.5.0:
+
+Version 1.5.0
+-------------
+The main focus of this release are improvements for the PDF generation with mapfish print, but it also other changes,
```

Comment:
missing word:
`... but it also other changes`  == should be something like ==> `... but it also contains other changes`

Reply:
changed, thanks